### ISSUE TITLE
Method calls in CDAClient builder has updated recently

### ISSUE THREAD

**Issue by davetao** [contentful/vault#64]
The example in the README.md needs to be updated in regards to setSpace and setToken

```
// Client
CDAClient client = new CDAClient.Builder()
    .setSpaceKey("cfexampleapi");
    .setAccessToken("b4c0n73n7fu1");
    .build();
```

**Comment by tomxor** [comment]
:+1: thanks, will reflect that in the README.

**Comment by tomxor** [comment]
Fixed via e03202e86e51f5b259278cc8c1b71d833fd3ce6c
